src/site/content/en/fast/identify-slow-third-party-javascript/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Identify slow third-party JavaScript
+subhead: Supercharge your performance detective skills with Lighthouse and Chrome DevTools.
+authors:
+  - mihajlija
+date: 2019-07-16
+# Add an updated date to your post if you edit in the future.
+# updated: 2019-06-27
+hero: hero.jpg
+alt: Inspecting a laptop with a magnifying glass.
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+description: |
+  Learn how to use Lighthouse and Chrome DevTools to identify slow third-party resources.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - fast
+---
+
+<style>
+  img[src*="icon"] {
+    max-width: 16px;
+    vertical-align: middle;
+  }
+</style>
+
+As a developer, you often don't have control over [which third-party scripts](/third-party-javascript/#network) your site loads. Before you can optimize third-party content you have to do some detective work to find out what's making your site slow. 🕵️
+
+In this post, you'll learn how to use [Lighthouse](https://developers.google.com/web/tools/lighthouse/) and [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/) to identify slow third-party resources. The post walks through increasingly robust techniques which are best used in combination.
+
+## If you only have 5 minutes
+
+The Lighthouse [Performance audit](/lighthouse-performance) helps you discover opportunities to speed up page loads. Slow third-party scripts are likely to appear in the **Diagnostics** section under the "Reduce JavaScript execution time" and "Avoid enormous network payloads" audits.
+
+To run an audit:
+
+{% Instruction 'devtools-audits', 'ol' %}
+1. Click **Mobile**.
+1. Select the **Performance** checkbox. (You can clear the rest of the checkboxes in the Audits section.)
+1. Click **Simulated Fast 3G, 4x CPU Slowdown**.
+1. Select the **Clear Storage** checkbox.
+1. Click **Run audits**.
+
+![Screenshot of the Chrome DevTools Audits panel.](./audits-panel.png)
+
+### Reduce JavaScript execution time
+The Lighthouse [JavaScript execution time](/bootup-time) audit highlights scripts that take a long time to parse, compile, or evaluate. Select the **Show 3rd-party resources** checkbox to discover CPU-intensive third-party scripts.
+
+![Screenshot showing that the "Show third-party resources" checkbox is checked.](show-third-party.png)
```

Comment:
Would you  mind switching this to an image tag and adding the `w-screenshot` class.
For anything that doesn't have a border or a strong drop shadow I think we tend to use the screenshot class so the image doesn't appear to be floating in the page.

Reply:
cc @mfriesenhahn as an fyi